docs/modules/faqs.html

```diff
@@ -181,6 +204,10 @@ <h3>Does Amazon Chime SDK support simulcast?</h3>
 						<h3>Does the SDK support remote mute where one attendee can mute another?</h3>
 					</a>
 					<p>Remote mute is the ability of one attendee in a meeting to mute another attendee. This feature is currently not supported server-side by the SDK. However, you can use <a href="https://aws.github.io/amazon-chime-sdk-js/modules/apioverview.html#9-send-and-receive-data-messages-optional">Amazon Chime SDK Data Messages</a> to send a message containing the attendee or attendees that should mute, and clients can mute themselves when they receive those messages.</p>
+					<a href="#when-there-are-multiple-monitors-plugged-in-my-screen-sharing-doesn39t-begin-unless-i-actively-interact-with-the-screen" id="when-there-are-multiple-monitors-plugged-in-my-screen-sharing-doesn39t-begin-unless-i-actively-interact-with-the-screen" style="color: inherit; text-decoration: none;">
+						<h3>When there are multiple monitors plugged in, my screen sharing doesn&#39;t begin unless I actively interact with the screen</h3>
+					</a>
+					<p>This seems to be a WebRTC issue and not specific to Chime SDK.</p>
```

Comment:
when we make a state like its WebRtc issue - can we back up with links or public references? Also is it a limitation of the specification or the underlying browser?